front-end/src/components/player-page/stats/stats-generators/DPMStatGenerator.ts

```diff
@@ -2,23 +2,16 @@ import {BaseStatGenerator} from "./BaseStatsGenerator";
 import PlayerStatModel from "../../../../../../Common/models/playerstat.model";
 
 export class DPMStatGenerator extends BaseStatGenerator {
-    getStatValue(playerStatsModels: PlayerStatModel[]): string {
-        let dmg = 0
-        let time = 0;
-        for (let playerStatsModel of playerStatsModels) {
-            dmg += playerStatsModel.totalDamageDealtToChampions;
-            time += playerStatsModel.gameLength;
-        }
-
-        let minutes = time / 60
-        return `${this.formatNumber(dmg/minutes)}` ;
-    }
-
     getStatTitle(): string {
         return "DPM"
     }
 
     getToolTip(): string {
         return "Damage Per Min";
     }
+
+    getStatValue(playerStatsModel: PlayerStatModel): number {
+        let minutes = playerStatsModel.gameLength / 60
```

Comment:
ig this one is probably ok, but maybe refactor to use the util function for riot gamelength to minutes. So incase for some reason this changes.